src/main/java/seedu/address/model/pet/ColorPattern.java

```diff
@@ -25,8 +25,8 @@ public ColorPattern(String colorPattern) {
      *
      * @return The color pattern.
      */
-    public String getColorPattern() {
-        return colorPattern;
+    public String getValue() {
```

Comment:
Can you update the method name in the Colorpattern test so that the test case will pass and CI will pass? thanks!